package.json

```diff
@@ -147,11 +153,26 @@
     "typescript": "^4.6.2",
     "vue-i18n-extract": "^2.0.0",
     "vue-jest": "^3.0.7",
-    "vue-tsc": "^0.34.7"
+    "vue-template-compiler": "^2.6.14",
+    "vue-tsc": "^0.34.7",
+    "webpack": "^4.46.0"
+  },
+  "pnpm": {
+    "peerDependencyRules": {
+      "ignoreMissing": [
+        "react",
+        "react-dom",
+        "consola"
+      ],
+      "allowedVersions": {
+        "babel-core": "7.0.0-bridge.0",
+        "postcss": "5.2.18"
+      }
+    }
```

Comment:
I was looking for something like this but I didn't know what to do with these "peerDependencies", nice!